### PR TITLE
Add ManyMC.app v0.0.7

### PR DESCRIPTION
**Pull request by dukecat0**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/manymc.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  zap trash: [
+    "~/Library/Application Support/ManyMC",
+    "~/Library/Preferences/org.multimc.ManyMC.plist",
+    "~/Library/Saved Application State/org.multimc.ManyMC.savedState",
+  ]
+
+  caveats do
+    depends_on_java "17+"
+  end
```
Comment (bevanjkay) [inline Casks/manymc.rb:28]: @meowmeowmeowcat I'm not sure that we should force a specific java version, as different MC versions/packs will require different java.

Comment (meowmeowmeowcat) [reply Casks/manymc.rb:28]: @bevanjkay It requires arm64 Java 17+ to run as [its README](https://github.com/MinecraftMachina/ManyMC#install) states.

Comment (meowmeowmeowcat) [reply Casks/manymc.rb:28]: I've checked and seems that you are right. I'll remove it. Thanks!

File: Casks/manymc.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  url "https://github.com/MinecraftMachina/ManyMC/releases/download/v#{version}/ManyMC.zip"
+  name "ManyMC"
+  desc "Familiar Minecraft Launcher with native support for arm64 (M1)"
+  homepage "https://github.com/MinecraftMachina/ManyMC"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
```
Comment (bevanjkay) [inline Casks/manymc.rb:13]: @meowmeowmeowcat The livecheck block should be able to be omitted as the default `:git` strategy will be inferred automatically from the url.

Comment (meowmeowmeowcat) [reply Casks/manymc.rb:13]: Fixed!

**Comment by bevanjkay** [comment]
Verified locally - won't run on CI due to no ARM64 tests being available.

**Pull request merged**